### ISSUE TITLE
A lot of empty lines are present in the generated SQL

### ISSUE THREAD

**Issue by lehphyro** [Backbase/blimp#2]
I've enabled the `stripComments` option and noticed that the generated SQL has a lot of empty lines in each SQL statement which makes SQL files very long.

Example plugin config: https://stash.backbase.com/projects/TRAN/repos/transaction-enrichment/browse/transaction-enricher/pom.xml?at=refs%2Fheads%2FMAINT-12225-many-locations-per-merchant#463-498

Example of generated SQL command:
```
CREATE TABLE location (


   id INTEGER GENERATED BY DEFAULT AS IDENTITY NOT NULL,


    external_id VARCHAR2(255) NOT NULL,


    latitude DECIMAL(10, 6),


    longitude DECIMAL(10, 6),


    address_line_1 VARCHAR2(255),


    address_line_2 VARCHAR2(255),


    post_code VARCHAR2(255),


    city VARCHAR2(255),


    state VARCHAR2(50),


    country VARCHAR2(255),


    complete_address VARCHAR2(255),


    source VARCHAR2(255) NOT NULL,


    inserted_at TIMESTAMP NOT NULL,


    updated_at TIMESTAMP,


    CONSTRAINT pk_location PRIMARY KEY (id)


);

ALTER TABLE merchant


   ADD CONSTRAINT fk_location_id


   FOREIGN KEY (location_id)


   REFERENCES location (id);

```

**Comment by pa314159** [comment]
Hey @lehphyro ,

The link is outdated, can you confirm the behaviour with the latest version of the plugin (0.16.4)?
Here is a configuration example: https://github.com/Backbase/golden-sample-services/blob/main/database/pom.xml

**Comment by lehphyro** [comment]
It was too long ago... maybe @ali-v-1985 can take this over together with you @pa314159 ?

**Comment by pa314159** [comment]
fixed in 0.16.5
